src/modules/blank-target-link.js

```diff
@@ -27,6 +27,7 @@
 				if (!!href && (/^https?:\/\//.test(href) || /^\/workspace/.test(href))) {
 					if (!t.attr('target')) {
 						t.attr('target', '_blank');
+						t.attr('rel', 'noopener');
 					}
```

Comment:
If there's no target attr set !!
We would not want to touch any links that do not have an href or are local links.